spec/services/arcgis_api/geocoder_spec.rb

```diff
@@ -5,7 +5,15 @@
 
   let(:subject) { ArcgisApi::Geocoder.new }
 
+  before(:each) do
+    Rails.cache.clear
```

Comment:
Looks like the cache is being cleared, but I'm not seeing any other references to `Rails.cache` in the diff? where are we using the cache?